### PR TITLE
[BEAM-1470] A quite logger for testing.

### PR DESCRIPTION
**Pull request by amitsela**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by amitsela** [comment]
R: @staslev

**Comment by staslev** [comment]
Consider this one, where we set the default to `ERROR` (so that only errors are printed out) but specifically set the `TestSparkRunner` to `INFO` so that test related stuff still show.
```
log4j.rootLogger=ERROR, testlogger

# A1 is set to be a ConsoleAppender.
log4j.appender.testlogger=org.apache.log4j.ConsoleAppender
log4j.appender.testlogger.target = System.err
log4j.appender.testlogger.layout=org.apache.log4j.PatternLayout
log4j.appender.testlogger.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n

log4j.logger.org.apache.beam.runners.spark.TestSparkRunner=INFO
```

**Comment by amitsela** [comment]
please retest this

File: runners/spark/src/test/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Set root logger level to ERROR.
+# set manually to INFO for debugging purposes.
+log4j.rootLogger=ERROR, testlogger
+
+# A1 is set to be a ConsoleAppender.
```
Comment (staslev) [inline runners/spark/src/test/resources/log4j.properties]: `A1`? 
Could be a leftover from previous stuff?

**Comment by staslev** [comment]
LGTM.

**Comment by jbonofre** [comment]
Sorry for the delay: LGTM too

**Pull request closed without merging**